skyportal/utils/offset.py

```diff
@@ -156,6 +157,101 @@ def get_ztfref_url(ra, dec, imsize, *args, **kwargs):
 }
 
 
+def calculate_best_position(
+    photometry, fallback=(None, None), how="snr", max_offset=0.5, sigma_clip=4.0
+):
+    """Calculates the best position for a source from its photometric
+       points
+
+    Parameters
+    ----------
+    photometry : list
+        List of Photometry objects
+    fallback : tuple, optional
```

Comment:
Why not simply raise an exception and let that propagate up to the calling function, rather than handle the exceptions inside this function?

Reply:
I'd prefer to have `calculate_best_position` always return a position rather than the possibility of raising an exception especially given all the ways this calc can go wrong.

Reply:
Renamed to make clear what this function is supposed to be used for.